src/components/ProfilePage/QuestCardCompleted/QuestCardCompleted.style.ts

```diff
@@ -68,31 +64,34 @@ export const QuestPlatformMainBox = styled(Box, {
   alignItems: 'center',
 }));
 
+export const QuestDatesBox = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  backgroundColor:
+    theme.palette.mode === 'light'
+      ? alpha(theme.palette.black.main, 0.04)
+      : theme.palette.alphaLight300.main,
+  paddingTop: '4px',
+  paddingBottom: '4px',
+  paddingLeft: '8px',
+  paddingRight: '8px',
+  borderRadius: '128px',
+  justifyContent: 'center',
+}));
+
 export interface XPDisplayBoxProps extends Omit<BoxProps, 'component'> {
   active?: boolean;
 }
 
 export const XPDisplayBox = styled(Box, {
   shouldForwardProp: (prop) => prop !== 'active',
 })<XPDisplayBoxProps>(({ active }) => ({
```

Comment:
If those suggestions make use of **theme** we must make sure, it´s provided. Just flying over it, I have not double-checked on each styled-component if theme is available!! ⚠️